data.sql

```diff
@@ -5,3 +5,13 @@ VALUES (2, 'Gabumon', '2018-11-15', 2, TRUE, 8.0),
 (3, 'Pikachu', '2021-07-01', 1, FALSE, 15.04),
 (4, 'Devimon', '2017-05-17', 5, TRUE, 11);
 
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
```

Comment:
-  Please, kindly ensure you're generating your IDs automatically instead of manually adding them as this might cause some errors in your database, and also, it would be hard for you to keep track of the numbers as you might make mistakes. it is best to generate them automatically. please try removing it.